### PR TITLE
Add long help with link to repo

### PR DESCRIPTION
**Pull request by maxbeizer**
I thought this could use with a (brief) long help description and link back to the repo for the docs.

WDYT @BlakeWilliams ?

### CODE REVIEW

File: internal/cmd/root.go [hunk 1]
```diff
@@ -9,8 +9,10 @@ import (
 )
 
 var rootCmd = &cobra.Command{
-	Use:   "rdm",
+	Use:   "remote-development-manager",
```
Comment (BlakeWilliams) [inline internal/cmd/root.go:12]:
```suggestion
	Use:   "rdm",
```

The binaries I share are generally named `rdm` which I don't think conflicts with other tools. Is there any reason we shouldn't use the short version here?

File: internal/cmd/root.go [hunk 2]
```diff
@@ -9,8 +9,10 @@ import (
...
 	Short: "A server and client for better remote development integration.",
+	Long: `Embetter your remove development experience!
+	Complete documentation is available at https://github.com/BlakeWilliams/remote-development-manager`,
```
Comment (BlakeWilliams) [inline internal/cmd/root.go:15]:
```suggestion
	Long: `Embetter your remote development experience!
	Complete documentation is available at https://github.com/BlakeWilliams/remote-development-manager`,
```

**Review by BlakeWilliams — APPROVED** [review]
I had two small comments, but otherwise LGTM!

File: internal/cmd/root.go (hunk 1 above)
Comment (maxbeizer) [reply internal/cmd/root.go:12]: Good point. 

What got me here is when you run `rdm` before installing the binary which probably doesn't make sense if you're seeing the help:

```
 $rdm
The program 'rdm' is currently not installed. You can install it by typing:
  brew install rtags
```

**Pull request merged**